packages/gestalt/src/Masonry.js

```diff
@@ -129,6 +129,7 @@ function statesForRendering<T>(props: Props<T>, state: State<T>) {
   const { items } = state;
 
   // Full layout is possible
+  // $FlowFixMe https://github.com/facebook/flow/issues/6151
```

Comment:
what are these flow errors and why weren't they there before?

Reply:
this is the exact flow error:
```
Missing type annotation for `T`.
```

not sure why it wasn't reported